From 309d11b4bbb402abdcf128920f353f4332f1943b Mon Sep 17 00:00:00 2001
From: Roman Li <Roman.Li@amd.com>
Date: Wed, 9 Apr 2025 12:03:00 -0400
Subject: drm/amd/display: Add htmldocs description for fused_io interface
Git-commit: 309d11b4bbb402abdcf128920f353f4332f1943b
Patch-mainline: v6.16-rc1
References: jsc#PED-13979 jsc#PED-14039 jsc#PED-14046 jsc#PED-14211 jsc#PED-14333 jsc#PED-14487 jsc#PED-14488 jsc#PED-14497 jsc#PED-14499 jsc#PED-15868

[Why]
htmldocs build warning: "Function parameter or struct member 'fused_io'
not described in 'amdgpu_display_manager'".

[How]
Add missing description.

Fixes: ce801e5d6c1b ("drm/amd/display: HDCP Locality check using DMUB Fused IO")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Roman Li <Roman.Li@amd.com>
Acked-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Acked-by: Patrik Jakobsson <pjakobsson@suse.de>

---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.h b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.h
index c8201a7a1ad6..4b615071e93b 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.h
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.h
@@ -639,6 +639,11 @@ struct amdgpu_display_manager {
 	 */
 	struct amdgpu_i2c_adapter *oem_i2c;
 
+	/**
+	 * @fused_io:
+	 *
+	 * dmub fused io interface
+	 */
 	struct fused_io_sync {
 		struct completion replied;
 		char reply_data[0x40];  // Cannot include dmub_cmd here
-- 
2.52.0

